recipes/linopy/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "linopy" %}
+{% set version = "0.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/linopy-{{ version }}.tar.gz
+  sha256: 78cc948d4a4840f6ce12337a75eaee79b6064d4517321ae0b1b495ca85455227
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8, <=3.10
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.7
+    - numpy
+    - scipy
+    - bottleneck
+    - toolz
+    - numexpr
+    - xarray <=2022.03.0
+    - dask-core >=0.18.0
+    - tqdm
+    - deprecation
+
+test:
+  imports:
+    - linopy
+  commands:
+    - pip check
+  requires:
+    - pip
+    - python >=3.8, <=3.10
+
+about:
+  home: https://github.com/PyPSA/linopy
+  summary: Linear optimization with N-D labeled arrays in Python
+  license: GPL-2.0-or-later
```

Comment:
Typo?
```suggestion
  license: GPL-3.0-or-later
```

Reply:
The bot wanted me to chose between [these SPDX identifiers](https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions)

Reply:
I'm not sure why we have that limited list, but that link has a link to the [full list of identifiers](https://spdx.org/licenses/) that includes the one I suggested. Since that's the actual version you have, let's use that. 😉

Reply:
alright :)